sdk/rust/oak_tests/src/lib.rs

```diff
@@ -36,6 +36,7 @@ pub fn compile_rust_wasm(cargo_path: &str, module_name: &str) -> std::io::Result
             "--target=wasm32-unknown-unknown",
             &format!("--manifest-path={}", cargo_path),
         ])
+        .env_remove("RUSTFLAGS")
```

Comment:
Perhaps we should just call [`env_clear`](https://doc.rust-lang.org/std/process/struct.Command.html#method.env_clear).

Reply:
`env_clear` causes wasm builds to fail, I'm not sure what entries are needed exactly though.